### ISSUE TITLE
React Native 0.29 Compatibility in Android

### ISSUE THREAD

**Issue by nicnocquee** [Microsoft/code-push#271]
Seems like `getPackages()` in [RN 0.29 Android](https://github.com/facebook/react-native/releases/tag/v0.29.0-rc.0) is not inside `MainActivity` anymore so the following doesn't compile anymore: 

```java

protected List<ReactPackage> getPackages() {
      return Arrays.<ReactPackage>asList(
          new MainReactPackage(),
          new CodePush(BuildConfig.CODEPUSH_KEY, this, BuildConfig.DEBUG),
      );
    }
```

`getPackages` is in `MainApplication` instead. Any suggestion on how to initiate the CodePush?

**Comment by lostintangent** [comment]
@nicnocquee There isn't currently a workaround to this. However, we have a [PR](https://github.com/Microsoft/react-native-code-push/pull/398) already out to add support to React Native 0.29-RC (as a goal, we try to add support to RN releases by the time they "RTM"), which should be merged in this afternoon, and we'll release it to NPM ASAP.

I'm going to close this as a dupe of https://github.com/Microsoft/react-native-code-push/issues/394, so that we can all track this via a single issue. Thanks for reaching out!

**Comment by lostintangent** [comment]
@nicnocquee We just released this fix to NPM, so if you upgrade to `1.13.0-beta` you should be good to go. [Here](https://github.com/Microsoft/react-native-code-push#plugin-configuration-android) are the docs for the RN 0.29-specific configuration steps. Let us know if you hit any issues. Thanks!
